src/dug/core/annotate.py

```diff
@@ -161,6 +161,12 @@ def expand_identifier(self, identifier, query_factory, kg_filename):
                 json.dump(response, stream, indent=2)
 
         # Get nodes in knowledge graph hashed by ids for easy lookup
+        noMessage = (len(response.get("message",{})) == 0)
+        statusError = (response.get("status","") == 'Error')
```

Comment:
The outer parentheses and default values in the `get` call are not strictly necessary and these can be simplified as 
```
noMessage = not response.get("message")
statusError = response.get("status") == 'Error'
```

Reply:
It's the influence of Go language creeping in.